### PR TITLE
Using debian Jessie base image instead of Wheezy

### PR DESCRIPTION
**Pull request by andriusa2**
* Uses Jessie base image for ISO construction
* Fixed some broken links on preseeded desktop to better correspond with current package versions.

### CODE REVIEW

File: config/chroot [hunk 1]
```diff
@@ -42,4 +42,4 @@ LB_UPDATES="true"
 
 # $LB_BACKPORTS: enable backports updates
 # (Default: false)
-LB_BACKPORTS="false"
+LB_BACKPORTS="true"
```
Comment (andriusa2) [inline config/chroot]: Need backports for codeblocks: https://packages.debian.org/search?keywords=codeblocks

File: config/package-lists/liox.list [hunk 2]
```diff
@@ -1,31 +1,22 @@
-# Desktop stuff
```
Comment (andriusa2) [inline config/package-lists/liox.list]: Comments in package lists break LB-4. Presumably to be fixed in LB-5.

File: config/chroot (hunk 1 above)
Comment (Motiejus) [inline config/chroot]: Can you add this comment to the file?

**Comment by Motiejus** [comment]
LGTM

**Comment by Motiejus** [comment]
Please also update the changelog after you merge this.

**Pull request merged**